CHANGELOG.md

```diff
@@ -1,6 +1,245 @@
+
+
+- **Repository moved to `WordPress\Requests`**
+
+  The `Requests` library has been officially adopted by the WordPress project
+  and has therefore been moved to `https://github.com/WordPress/Requests`.
+
+  All links in code and documentation were updated accordingly.
+
+  (props [@dd32][gh-dd32], [@JustinyAhin][gh-JustinyAhin], [@jrfnl][gh-jrfnl], [@rmccue][gh-rmccue], [#440][gh-440], [#441][gh-441], [#448][gh-448])
+
+
+- **Manage `"Expect"` header with `cURL` transport**
+
+  By default, `cURL` adds a `Expect: 100-Continue` header to certain requests. This can add as much as a second delay to requests done using `cURL`. This is [discussed on the cURL mailing list](https://curl.se/mail/lib-2017-07/0013.html).
+
+  To prevent this, `Requests` now adds an empty `"Expect"` header to requests that are smaller than 1 MB and use HTTP/1.1.
+
+  (props [@carlalexander][gh-carlalexander], [@schlessera][gh-schlessera], [@TimothyBJacobs][gh-TimothyBJacobs], [#453][gh-453], [#454][gh-454], [#469][gh-469])
+
+
+- **Update bundled certificates as of 2021-02-12**
+
+  The bundled certificates were updated. A small subset of expired certificates are still included for legacy reasons (and support).
+
+  (props [@ozh][gh-ozh], [@patmead][gh-patmead], [@schlessera][gh-schlessera], [@todeveni][gh-todeveni], [#385][gh-385], [#398][gh-398], [#451][gh-451])
+
+
+- **Add required `Content-*` headers for empty `POST` requests**
+
+  Sends the `Content-Length` and `Content-Type` headers even for empty `POST` requests, as the length is expected as per [RFC2616 Section 14.13](https://tools.ietf.org/html/rfc2616#section-14.13):
+  ```
+  Content-Length header "SHOULD" be included. In practice, it is not
+  used for GET nor HEAD requests, but is expected for POST requests.
+  ```
+
+  (props [@dd32][gh-dd32], [@gstrauss][gh-gstrauss], [@jrfnl][gh-jrfnl], [@soulseekah][gh-soulseekah], [#248][gh-248], [#249][gh-249], [#318][gh-318], [#368][gh-368])
+
+
+- **Ignore locale when creating the HTTP version string from a float**
+
+  The previous behavior allowed for the locale to mess up the float to string conversion resulting in a `GET / HTTP/1,1` instead of `GET / HTTP/1.1` request.
+
+  (props [@tonebender][gh-tonebender], [@Zegnat][gh-Zegnat], [#335][gh-335], [#339][gh-339])
+
+
+- **Make `verify => false` work with `fsockopen`**
+
+  This allows the `fsockopen` transport now to ignore SSL failures when requested.
+  
+  (props [@soulseekah][gh-soulseekah], [#310][gh-310], [#311][gh-311])
+
+
+- **Only include port number in the `Host` header if it differs from the default**
+
+  The code was not violating the RFC per se, but also not following standard practice of leaving the port off when it is the default port for the scheme, which could lead to connectivity issues.
+
+  (props [@amandato][gh-amandato], [@dd32][gh-dd32], [#238][gh-238])
```

Comment:
amandato is not yet in the list at the bottom of the changelog. Link needs to be added: https://github.com/amandato